apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -245,6 +245,33 @@ function ListBoxInline({ options, layout }) {
     containerPadding = layoutOptions.layoutOrder === 'row' ? '2px 4px' : '2px 6px 2px 4px';
   }
 
+  const SearchIconComp = constraints?.active ? (
+    <SearchIcon title={translator.get('Listbox.Search')} size="large" style={{ fontSize: '12px', padding: '7px' }} />
+  ) : (
+    <IconButton
+      onClick={onShowSearch}
+      tabIndex={-1}
+      title={translator.get('Listbox.Search')}
+      size="large"
+      disableRipple
+      data-testid="search-toggle-btn"
+    >
+      <SearchIcon style={{ fontSize: '12px' }} />
+    </IconButton>
+  );
+
+  const LockIconComp = selectDisabled() ? (
+    <Lock
+      title={translator.get('SelectionToolbar.ClickToUnlock')}
```

Comment:
maybe change the tooltip in the case. as you can not click it

Reply:
Removed it. There is no tooltip for lock icon in edit mode for old filter pane.